.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: required
```

Comment:
That might be a problem with coverage but anyway I don't want to limit forks.

Not good but better that was - leave it!